Demo2.js

```diff
@@ -3,6 +3,12 @@
 import React from 'react';
 import Springs from './Springs';
 
+if (!String.prototype.includes) {
+  String.prototype.includes = function() {'use strict';
+    return String.prototype.indexOf.apply(this, arguments) !== -1;
+  };
+}
+
```

Comment:
https://babeljs.io/docs/learn-es2015/#math-number-string-object-apis

Babel polyfills these already, we don't need this =)